src/Scenes/GameScene/GameScene.ts

```diff
@@ -49,6 +49,10 @@ export class GameScene implements Scene {
 			this._session.runTurn(PlayerAction.MoveDown);
 		} else if (this._game.rawInput.isKeyPressed('3')) {
 			this._session.runTurn(PlayerAction.MoveDownRight);
+		} else if (this._game.rawInput.isKeyPressed('r')) {
+			const player = this._session.level.entities.getPlayer();
+			player && this._session.registerRecording(player.movesQueue.copy());
+			this._session.resetLevel();
```

Comment:
I feel like these should be moved into a separate method? As restting the level might be done in more ways than just pressing r (e.g. death)

Reply:
I think you're right, but I'd rather wait until we need it somewhere else to see how it fits rather than arbitrarily decide at the moment.